owtf/plugin/plugin_helper.py

```diff
@@ -38,49 +38,65 @@ def __init__(self):
         self.shell = self.get_component("shell")
         self.timer = self.get_component("timer")
         # Compile regular expressions only once on init:
-        self.RobotsAllowRegexp = re.compile("Allow: ([^\n  #]+)")
-        self.RobotsDisallowRegexp = re.compile("Disallow: ([^\n #]+)")
-        self.RobotsSiteMap = re.compile("Sitemap: ([^\n #]+)")
-
-    def MultipleReplace(self, Text, ReplaceDict):  # This redundant method is here so that plugins can use it
-        return MultipleReplace(Text, ReplaceDict)
-
-    def CommandTable(self, Command):
+        self.robots_allow_regex = re.compile("Allow: ([^\n  #]+)")
+        self.robots_disallow_regex = re.compile("Disallow: ([^\n #]+)")
+        self.robots_sitemap = re.compile("Sitemap: ([^\n #]+)")
+
+    def multi_replace(self, text, replace_dict):
```

Comment:
I am not refactoring any of this. This should never be done on the server side. Will be ported over to the React app.